packages/client/src/pages/admin/pluggables/Reports.tsx

```diff
@@ -132,21 +153,19 @@ export const ReportsWidget = withFetchApi(
 							<div>{this.state.error}</div>
 						) : (
 							<div>
-								{(this.props.account.type === AccountType.CAPSQUADRON ||
+								{/* {(this.props.account.type === AccountType.CAPSQUADRON ||
 									this.props.account.type === AccountType.CAPEVENT) &&
-								hasPermission('PromotionManagement')(
-									Permissions.PromotionManagement.FULL,
-								)(this.props.member) ? (
-									<>
-										<Button
-											onClick={() => this.createSQR601()}
-											buttonType="none"
-										>
-											SQR 60-1 Cadet status report
-										</Button>
-										<br />
-									</>
-								) : null}
+									(
+										hasPermission('PromotionManagement')(Permissions.PromotionManagement.FULL)(this.props.member) ||
+										hasAllowedDutyPosition(this.props.member)
+									) ? ( */}
```

Comment:
Same as before

Reply:
It's a placeholder.  I'm not done with it yet.  I'll come back to it.